res/values-ga/strings.xml

```diff
@@ -72,14 +72,14 @@
     <string name="Max_Players">"Rannpháirtithe Max"</string>
     <string name="Min_Players">"Rannpháirtithe Min"</string>
 
-    <string name="OK">"OK"</string>
+    <string name="OK">"ceart go leor"</string>
```

Comment:
Please keep the capital letters here; It seems to be too long length. I am afraid this is not correct translation.